kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/TestEngine.kt

```diff
@@ -57,8 +57,8 @@ class TestEngine(private val config: TestEngineConfig) {
                config.configuration.concurrentSpecs ?: config.configuration.parallelism,
                context,
             )
-            Platform.JS -> SequentialTestSuiteScheduler(context)
-            Platform.Native -> SequentialTestSuiteScheduler(context)
+
+            else -> SequentialTestSuiteScheduler(context)
```

Comment:
I like keeping exhaustive whens, forces us to come back here if we ever need to add another platform.
```suggestion

            Platform.JS,
            Platform.Native,
            Platform.WasmJs -> SequentialTestSuiteScheduler(context)
```

Reply:
Fine with me.